### ISSUE TITLE
Mask Decoder finetuning for SAMed_h

### ISSUE THREAD

**Issue by NathanMalta** [hitachinsk/SAMed#11]
Thanks for your work on this project, it's very interesting!  I noticed that you don't have an option to finetune the mask decoder for SAMed_h.  Is there a theoretical or technical reason behind this? VRAM constraints? 

Just curious as SAMed_h is already trading blows with your Table 1 baselines so perhaps SAMed_h + mask decoder finetuning could be fully SoTA.

Thanks again!

**Comment by hitachinsk** [comment]
I have checked the codes of SAMed_h carefully. I only freeze the parameters of the image encoder, and the fine-tuning of the parameters of mask decoder is activated by default. Why do you argue I did not finetune the mask decoder? Could you provide more information?

**Comment by NathanMalta** [comment]
You are correct, I misunderstood the code.  Thank you for the clarification.

On another topic, you mention on page 11 that LoRA finetuning reduces the performance of the mask decoder when compared to full finetuning.  Do you feel that this effect would be as large for LoRA vs full funetuning of the image encoder? Thank you

**Comment by hitachinsk** [comment]
Maybe. But I did not do such experiment because the computation cost is too big. And the deployment and storage cost is also unaffordable in practical usage.
